third_party/amd/lib/TritonAMDGPUToLLVM/Utility.h

```diff
@@ -22,6 +22,7 @@ const char predicatedStore[] = "__predicated_store";
 const char predicatedStoreCG[] = "__predicated_store_CG";
 const char predicatedStoreCS[] = "__predicated_store_CS";
 const char predicatedStoreWT[] = "__predicated_store_WT";
+const char noAliasAsyncLoads[] = "__no_alias_async_loads";
```

Comment:
While we are here, can we prefix L17-L24 with `__triton_hip_` to make it clear? (These are just special strings we use inside Triton so better to avoid confusion. Probably we need to define some amdgpu helper ops to make it more principled instead of doing string matching, but later.)